rfcs/text/0037/host.yml

```diff
@@ -0,0 +1,193 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+
+### Overview ###
+# 
+# The following metrics give an overview on the current state of a host. On a high level it collects
+# the following data:
+#
+# * CPU used (in %) and load
+# * Memory used (in %, used, total)
+# * Disk usage (in %)  and io -> summary
+# * Network (traffic in / out)
+#
+# This translates to the detailed metric list below.
+
+
+### CPU ###
+
+# The CPU metrics must indicate under how much load the system is. 
```

Comment:
Would we need any guidance how these new proposed `host.cpu.*` fields are related to the existing [`host.cpu.usage`](https://github.com/elastic/ecs/blob/main/schemas/host.yml#L122)?

Reply:
Yes. I added this to the list of concerns.